charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -217,7 +217,7 @@ data:
                 - "${SQS_QUEUE_URL}"
                 {{- end }}
                 - --num-workers
-                - "${PER_WORKER}"
+                - "${FORWARDER_WORKER_COUNT}"
```

Comment:
we always just have one celery worker and `per_worker` is used to determine the concurrency https://github.com/scaleapi/llm-engine/blob/main/model-engine/model_engine_server/inference/forwarding/celery_forwarder.py#L141

Reply:
imo we could think of ways to maybe increase the number of celery tasks in flight to increase async task throughput, which may involve increasing number of workers, maybe not super necessary now though